src/enclave/enclave_init.c

```diff
@@ -150,9 +156,8 @@ int __libc_init_enclave(int argc, char** argv)
     max_lthreads = next_power_of_2(max_lthreads);
 
     newmpmcq(&__scheduler_queue, max_lthreads, 0);
-
-    __init_libc(envp, argv[0]);
-    __init_tls();
```

Comment:
Just double checking. Has the call to this function gone away completely? (__init_tls)?